rules_haskell_tests/tests/RunTests.hs

```diff
@@ -149,10 +149,12 @@ main = hspec $ do
   it "bazel test examples" $ do
     assertSuccess $ (bazel ["build", "//..."]) { Process.cwd = Just "../examples" }
     assertSuccess $ (bazel ["test", "//..."]) { Process.cwd = Just "../examples" }
+    assertSuccess $ (bazel ["shutdown"]) { Process.cwd = Just "../examples" }
```

Comment:
Maybe we also need to shutdown the bazel server from the previous spec items?

Reply:
Agreed. I am not sure where I should put the shutdown. Not being familiar with Haskell or the test framework, I am not sure if the tests and test cases are executed in order from top to bottom. That is on my list of things to do today. If you have a suggestion on where to put the shutdown, please let me know.

Reply:
Probably the easiest option is to shutdown the server before starting the `examples` and `tutorial` run? E.g.
```
it "bazel test examples" $ do
    assertSuccess $ bazel ["shutdown"]
    assertSuccess $ (bazel ["build", "//..."]) { Process.cwd = Just "../examples" }
    assertSuccess $ (bazel ["test", "//..."]) { Process.cwd = Just "../examples" }
    assertSuccess $ (bazel ["shutdown"]) { Process.cwd = Just "../examples" }
```
likewise for the tutorial. This way it does not matter in which order the items are run and you can also run individual items if you want.

Reply:
Good point. I will add that with comments explaining why we added these.